pkg/pgmodel/migrations/sql/idempotent/base.sql

```diff
@@ -483,6 +491,17 @@ $$
 LANGUAGE SQL VOLATILE;
 GRANT EXECUTE ON FUNCTION SCHEMA_CATALOG.get_or_create_label_key_pos(text, text) to prom_writer;
 
+-- label_cardinality returns the cardinality of a label_pair id in the series table.
+-- In simple terms, it means the number of times a label_pair/label_matcher is used
+-- across all the series.
+CREATE OR REPLACE FUNCTION SCHEMA_CATALOG.label_cardinality(label_id INT)
```

Comment:
```suggestion
CREATE OR REPLACE FUNCTION SCHEMA_PROM.label_cardinality(label_id INT)
```
since this is a public function